lib/src/theme/themes/default_theme_no_secondary_border_variant.dart

```diff
@@ -237,6 +238,46 @@ class ShadDefaultThemeNoSecondaryBorderVariant extends ShadThemeVariant {
     );
   }
 
+  @override
+  ShadBreadcrumbTheme breadcrumbTheme() {
+    return ShadBreadcrumbTheme(
+      separator: Icon(
+        LucideIcons.chevronRight,
+        size: 14,
+        color: colorScheme.mutedForeground,
+      ),
```

Comment:
```suggestion
         separator: const ShadBreadcrumbSeparator(),
```

OR we can leave it `null`.
Also we need to update `ShadBreadcrumbTheme` in default_theme_variant.dart.